include/odp/api/spec/ipsec.h

```diff
@@ -1626,6 +1627,18 @@ int odp_ipsec_sa_mtu_update(odp_ipsec_sa_t sa, uint32_t mtu);
  */
 void *odp_ipsec_sa_context(odp_ipsec_sa_t sa);
 
+/**
+ * Print IPSEC SA info
+ *
+ * @param sa      SA handle
+ * @param fp      Pointer to a file for output
+ *
+ * Print implementation-defined IPSEC SA debug information to the log file.
+ * The implementation may choose to print global IPSEC configuration when the
+ * SA handle passed is ODP_IPSEC_SA_INVALID.
+ */
+void odp_ipsec_sa_print(odp_ipsec_sa_t sa, FILE *fp);
```

Comment:
Other similar print functions of ODP API do not include FILE parameter. It should be remove also here and just state that function prints into ODP log (global init has log print function as parameter).

Also remove optional behavior of ODP_IPSEC_SA_INVALID. If IPSEC global config, etc debug information is needed, then it is much more clear to introduce another debug function for that purpose. So, this should just print SA specific info.

Reply:
Agreed. In V2, removed the FILE parameter and introduced a separate odp_ipsec_print() API to print the global config information.